data/tutorials/guides/1wf_00_compiling_ocaml_projects.md

```diff
@@ -177,6 +227,70 @@ guide](https://dune.readthedocs.io/en/latest/quick-start.html) shows you how to
 write such description files for more complicated situations, and how to
 structure, build, and run dune projects.
 
+### Bytecode Using Dune
+
+Dune is a build system for OCaml projects, and it allows you to configure different modes for building your executables. We will use `(modes byte exe)` stanza in the `dune` file, which will produce both bytecode (interpreted) and native executable versions of the OCaml program. 
+
+Let's create an example project named `myproject`.
+
+```dune
+mkdir myproject
+cd myproject
+opam init
+```
+
+`Note:` *opam init* command might prompt you to confirm the initialization process and may also provide instructions to add Opam-related lines to your shell configuration file (~/.bashrc or ~/.profile). Following any instructions provided is important to ensure Opam is properly integrated into your shell environment.
+
+Create a `dune-project` file, add the following content.
+
+```dune
+(lang dune 3.0)
+(name myproject)
+```
+
+Here, 3.0 is the installed version of `Dune`. You can check it by typing `dune --version` on your terminal. And the `name` is the name of the project, i.e., `myproject`.
+
+Create a `dune` file, add the following content.
+
+```dune
+(executable
+ (name main)
+ (libraries base)
+ (modes byte exe))
+```
+
+As aforementioned, `(modes byte exe)` stanza produces both bytecode (interpreted) and native executable versions of our OCaml program.
+
+Next, create a `main.ml` file, add the following content.
+
+```dune
+let () = print_endline "Hello Dune!"
+```
+
+Finally, we compile and execute it.
+
+```dune
+dune build main.bc
+```
+
+The `.bc` stands for generic bytecode file, and it can be an executable or library.
+
+
+```dune
+dune exec ./main.bc
+```
+
+Now `Hello Dune!` will be printed on the screen.
+
+We can also do that with `.exe`.
+
+```dune
+dune build main.exe
+```
+
+```dune
+dune exec ./main.exe
+```
```

Comment:
````suggestion
```shell
# dune build main.exe

# dune exec ./main.exe
Hello Dune!                      
```
````

Reply:
Will **$** work instead of **#**?

Reply:
We use $ in the other tutorials. We should use it here as well.